configapp/configapp.py

```diff
@@ -18,6 +18,12 @@
 predict_dir = os.path.join(filedir, app.config['PREDICTION_PATH'])
 
 
+@app.context_processor
+def inject_picture_size():
```

Comment:
@mrshu Yes, that does make sense, but currently I don't see any reason to pass such arguments to frontend. If we decide we want to handle more arguments in frontend, I will create js class available to entire frontend application.